common/include/vulkan_interfaces.h

```diff
@@ -19,10 +19,21 @@
 #define VK_ENABLE_BETA_EXTENSIONS 1
 #endif
 
-// This flag is defined in CTS.
+// This flag is defined in CTS to build the parser source files which are also in the vk namespace in external/vulkancts/modules/vulkan/video
 #ifdef DE_BUILD_VIDEO
 #include "vkDefs.hpp"
 using namespace vk;
+// This flag is defined in CTS to build VVS library and provide the vulkan definitions from CTS in vkVulkan_c.inl
+#elif defined(VVS_CTS_VULKAN_HEADER)
+#include "vkDefs.h"
+#if defined(VK_USE_PLATFORM_WIN32_KHR)
```

Comment:
Overall I'm OK with the changes.

Stupid question: what happens if you just do

``` 
#undef VK_USE_PLATFORM_WIN32_KHR
``` 

??

As a personal comment about coding taste: I'm not a big fan of juggling with symbols in code. I prefer to decouple the presentation layer with the video processing, so the cmake scripts would decide which files to include in the compilation.

Reply:
Thanks for your concern and I agree with your position about the extra defines used. So I came back to my initial solution which was not working because of the mssing windows.h include. So now VK_USE_PLATFORM_WIN32_KHR is used only for Vulkan purpose. Otherwise I changed it with _WIN32 guard which is more relevant. Now VK_USE_PLATFORM_WIN32_KHR wont be defined in CTS.